teste/contratogeral.http

```diff
@@ -0,0 +1,34 @@
+GET http://localhost:3000/contratogeral 
+###
+GET http://localhost:3000/contratogeral/1
+###
+GET http://localhost:3000/contratogeral/Ricardo%20Chaves
+
+###
+POST http://localhost:3000/contratogeral
+Content-Type: application/json
+
+{
+    "descricao_servico" : "Manutenção da Rede Elétrica",
+    "responsavel" : "Fernando Jorge",
+    "data_inicio" : "2024-12-03T10:30:00.000Z",
+    "data_fim" : "2024-12-13T10:30:00.000Z",
+    "valor" : 100000.00,
+    "prestadorId" : 1,
+    "clienteId" : 2,
+    "servicoId" : 3
+   
+}
+
+###
+DELETE http://localhost:3000/contratogeral/4
+
+###
+PUT http://localhost:3000/contratogeral/4
+Content-Type: application/json
+
+{
+    "descricao_servico" : "Manutenção da Rede Elétrica",
+    "responsavel" : "Ricardo Chaves Jr",
+    "valor": 120000.00
+}
```

Comment:
Se for alterar para `contrato-geral`, lembrar de trocar os testes, bem como o GET `/contrato-geral/responsavel/Ricardo%20Chaves`